From: Stefano Rivera <stefanor@debian.org>
Date: Tue, 18 Jul 2023 18:49:22 +0200
Subject: Ansible 2.14 support

Anecdotal evidence on the upstream issue suggests that there are no
changes required.
See also: https://github.com/mitogen-hq/mitogen/pull/977

Bug-Upstream: https://github.com/mitogen-hq/mitogen/issues/974
Forwarded: not-needed
---
 ansible_mitogen/loaders.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ansible_mitogen/loaders.py b/ansible_mitogen/loaders.py
index 1f4d8fc..754ec6c 100644
--- a/ansible_mitogen/loaders.py
+++ b/ansible_mitogen/loaders.py
@@ -49,7 +49,7 @@ __all__ = [
 
 
 ANSIBLE_VERSION_MIN = (2, 10)
-ANSIBLE_VERSION_MAX = (2, 13)
+ANSIBLE_VERSION_MAX = (2, 14)
 
 NEW_VERSION_MSG = (
     "Your Ansible version (%s) is too recent. The most recent version\n"
